21/08/2024 11:46:24|Warn |Logger.set_Level|The current logging level has been changed to Debug.
21/08/2024 11:46:24|Fatal|WebSocket.connect|WebSocketSharp.WebSocketException: An error has occurred during a TLS handshake. ---> System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Mono.Security.Interface.TlsException: Handshake failed - error code: UNITYTLS_INTERNAL_ERROR, verify result: UNITYTLS_X509VERIFY_NOT_DONE
                            at Mono.Unity.Debug.CheckAndThrow (Mono.Unity.UnityTls+unitytls_errorstate errorState, Mono.Unity.UnityTls+unitytls_x509verify_result verifyResult, System.String context, Mono.Security.Interface.AlertDescription defaultAlert) [0x00036] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Unity.UnityTlsContext.ProcessHandshake () [0x00082] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake (Mono.Net.Security.AsyncOperationStatus status, System.Boolean renegotiate) [0x000da] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake(Mono.Net.Security.AsyncOperationStatus,bool)
                            at Mono.Net.Security.AsyncHandshakeRequest.Run (Mono.Net.Security.AsyncOperationStatus status) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.AsyncProtocolRequest.ProcessOperation (System.Threading.CancellationToken cancellationToken) [0x000fc] in <e2f37d58276e431f93af9ff200e09912>:0 
                             --- End of inner exception stack trace ---
                            at Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x0004b] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient(string,System.Security.Cryptography.X509Certificates.X509CertificateCollection,System.Security.Authentication.SslProtocols,bool)
                            at System.Net.Security.SslStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at WebSocketSharp.WebSocket.setClientStream () [0x000e1] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                             --- End of inner exception stack trace ---
                            at WebSocketSharp.WebSocket.setClientStream () [0x000fb] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.doHandshake () [0x00000] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.connect () [0x00073] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
21/08/2024 11:46:24|Debug|WebSocket.closeHandshake|Was clean?: False
                            sent: False
                            received: False
21/08/2024 11:46:24|Debug|WebSocket.sendHttpRequest|A request to the server:
                          GET /?userId=66aa283a7093a0984b7188e5 HTTP/1.1
                          User-Agent: websocket-sharp/1.0
                          Host: retstudio.io.vn
                          Upgrade: websocket
                          Connection: Upgrade
                          Sec-WebSocket-Key: wlpMnCl7tRdr78d2ndn2jQ==
                          Sec-WebSocket-Version: 13
21/08/2024 11:46:24|Debug|WebSocket.sendHttpRequest|A response to this request:
                          HTTP/1.1 101 Switching Protocols
                          Date: Wed, 21 Aug 2024 04:46:24 GMT
                          Connection: upgrade
                          Upgrade: websocket
                          Sec-WebSocket-Accept: qMGN8fsnhgjOn0nTlXEG7j1hLP0=
                          CF-Ray: 8b67faad0d29848b-HKG
                          CF-Cache-Status: DYNAMIC
                          Server: cloudflare
                          alt-svc: h3=":443"; ma=86400
21/08/2024 11:47:00|Info |WebSocket.close|The closing is already in progress.
21/08/2024 11:47:00|Debug|WebSocket.closeHandshake|Was clean?: True
                            sent: True
                            received: True
21/08/2024 11:47:00|Info |WebSocket.close|The connection has already been closed.
21/08/2024 11:47:00|Info |WebSocket.close|The connection has already been closed.
21/08/2024 12:27:14|Warn |Logger.set_Level|The current logging level has been changed to Debug.
21/08/2024 12:27:14|Fatal|WebSocket.connect|WebSocketSharp.WebSocketException: An error has occurred during a TLS handshake. ---> System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Mono.Security.Interface.TlsException: Handshake failed - error code: UNITYTLS_INTERNAL_ERROR, verify result: UNITYTLS_X509VERIFY_NOT_DONE
                            at Mono.Unity.Debug.CheckAndThrow (Mono.Unity.UnityTls+unitytls_errorstate errorState, Mono.Unity.UnityTls+unitytls_x509verify_result verifyResult, System.String context, Mono.Security.Interface.AlertDescription defaultAlert) [0x00036] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Unity.UnityTlsContext.ProcessHandshake () [0x00082] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake (Mono.Net.Security.AsyncOperationStatus status, System.Boolean renegotiate) [0x000da] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake(Mono.Net.Security.AsyncOperationStatus,bool)
                            at Mono.Net.Security.AsyncHandshakeRequest.Run (Mono.Net.Security.AsyncOperationStatus status) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.AsyncProtocolRequest.ProcessOperation (System.Threading.CancellationToken cancellationToken) [0x000fc] in <e2f37d58276e431f93af9ff200e09912>:0 
                             --- End of inner exception stack trace ---
                            at Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x0004b] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient(string,System.Security.Cryptography.X509Certificates.X509CertificateCollection,System.Security.Authentication.SslProtocols,bool)
                            at System.Net.Security.SslStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at WebSocketSharp.WebSocket.setClientStream () [0x000e1] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                             --- End of inner exception stack trace ---
                            at WebSocketSharp.WebSocket.setClientStream () [0x000fb] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.doHandshake () [0x00000] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.connect () [0x00073] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
21/08/2024 12:27:14|Debug|WebSocket.closeHandshake|Was clean?: False
                            sent: False
                            received: False
21/08/2024 12:27:14|Debug|WebSocket.sendHttpRequest|A request to the server:
                          GET /?userId=66aa283a7093a0984b7188e5 HTTP/1.1
                          User-Agent: websocket-sharp/1.0
                          Host: retstudio.io.vn
                          Upgrade: websocket
                          Connection: Upgrade
                          Sec-WebSocket-Key: vb3dXLtIw5wvG6ac5V1zhw==
                          Sec-WebSocket-Version: 13
21/08/2024 12:27:15|Debug|WebSocket.sendHttpRequest|A response to this request:
                          HTTP/1.1 101 Switching Protocols
                          Date: Wed, 21 Aug 2024 05:27:15 GMT
                          Connection: upgrade
                          Upgrade: websocket
                          Sec-WebSocket-Accept: 0p5a9XTFh52VK51JSFvRBOK3Uok=
                          CF-Ray: 8b68367f5dbe8502-HKG
                          CF-Cache-Status: DYNAMIC
                          Server: cloudflare
                          alt-svc: h3=":443"; ma=86400
21/08/2024 12:30:31|Info |WebSocket.close|The closing is already in progress.
21/08/2024 12:30:31|Debug|WebSocket.closeHandshake|Was clean?: True
                            sent: True
                            received: True
21/08/2024 12:30:31|Info |WebSocket.close|The connection has already been closed.
21/08/2024 12:30:31|Info |WebSocket.close|The connection has already been closed.
21/08/2024 12:37:40|Warn |Logger.set_Level|The current logging level has been changed to Debug.
21/08/2024 12:37:40|Fatal|WebSocket.connect|WebSocketSharp.WebSocketException: An error has occurred during a TLS handshake. ---> System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Mono.Security.Interface.TlsException: Handshake failed - error code: UNITYTLS_INTERNAL_ERROR, verify result: UNITYTLS_X509VERIFY_NOT_DONE
                            at Mono.Unity.Debug.CheckAndThrow (Mono.Unity.UnityTls+unitytls_errorstate errorState, Mono.Unity.UnityTls+unitytls_x509verify_result verifyResult, System.String context, Mono.Security.Interface.AlertDescription defaultAlert) [0x00036] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Unity.UnityTlsContext.ProcessHandshake () [0x00082] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake (Mono.Net.Security.AsyncOperationStatus status, System.Boolean renegotiate) [0x000da] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake(Mono.Net.Security.AsyncOperationStatus,bool)
                            at Mono.Net.Security.AsyncHandshakeRequest.Run (Mono.Net.Security.AsyncOperationStatus status) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.AsyncProtocolRequest.ProcessOperation (System.Threading.CancellationToken cancellationToken) [0x000fc] in <e2f37d58276e431f93af9ff200e09912>:0 
                             --- End of inner exception stack trace ---
                            at Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x0004b] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient(string,System.Security.Cryptography.X509Certificates.X509CertificateCollection,System.Security.Authentication.SslProtocols,bool)
                            at System.Net.Security.SslStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at WebSocketSharp.WebSocket.setClientStream () [0x000e1] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                             --- End of inner exception stack trace ---
                            at WebSocketSharp.WebSocket.setClientStream () [0x000fb] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.doHandshake () [0x00000] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.connect () [0x00073] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
21/08/2024 12:37:40|Debug|WebSocket.closeHandshake|Was clean?: False
                            sent: False
                            received: False
21/08/2024 12:37:41|Debug|WebSocket.sendHttpRequest|A request to the server:
                          GET /?userId=66aa283a7093a0984b7188e5 HTTP/1.1
                          User-Agent: websocket-sharp/1.0
                          Host: retstudio.io.vn
                          Upgrade: websocket
                          Connection: Upgrade
                          Sec-WebSocket-Key: nV2gp1ScLmvb2LprBejkjw==
                          Sec-WebSocket-Version: 13
21/08/2024 12:37:41|Debug|WebSocket.sendHttpRequest|A response to this request:
                          HTTP/1.1 101 Switching Protocols
                          Date: Wed, 21 Aug 2024 05:37:41 GMT
                          Connection: upgrade
                          Upgrade: websocket
                          Sec-WebSocket-Accept: z2IclGpIv9JbGchYBOZfBQ/nshw=
                          CF-Ray: 8b6845c8e8c68502-HKG
                          CF-Cache-Status: DYNAMIC
                          Server: cloudflare
                          alt-svc: h3=":443"; ma=86400
21/08/2024 12:38:35|Info |WebSocket.close|The closing is already in progress.
21/08/2024 12:38:35|Debug|WebSocket.closeHandshake|Was clean?: True
                            sent: True
                            received: True
21/08/2024 12:38:35|Info |WebSocket.close|The connection has already been closed.
21/08/2024 12:38:35|Info |WebSocket.close|The connection has already been closed.
21/08/2024 12:40:35|Warn |Logger.set_Level|The current logging level has been changed to Debug.
21/08/2024 12:40:35|Fatal|WebSocket.connect|WebSocketSharp.WebSocketException: An error has occurred during a TLS handshake. ---> System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Mono.Security.Interface.TlsException: Handshake failed - error code: UNITYTLS_INTERNAL_ERROR, verify result: UNITYTLS_X509VERIFY_NOT_DONE
                            at Mono.Unity.Debug.CheckAndThrow (Mono.Unity.UnityTls+unitytls_errorstate errorState, Mono.Unity.UnityTls+unitytls_x509verify_result verifyResult, System.String context, Mono.Security.Interface.AlertDescription defaultAlert) [0x00036] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Unity.UnityTlsContext.ProcessHandshake () [0x00082] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake (Mono.Net.Security.AsyncOperationStatus status, System.Boolean renegotiate) [0x000da] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake(Mono.Net.Security.AsyncOperationStatus,bool)
                            at Mono.Net.Security.AsyncHandshakeRequest.Run (Mono.Net.Security.AsyncOperationStatus status) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.AsyncProtocolRequest.ProcessOperation (System.Threading.CancellationToken cancellationToken) [0x000fc] in <e2f37d58276e431f93af9ff200e09912>:0 
                             --- End of inner exception stack trace ---
                            at Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x0004b] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient(string,System.Security.Cryptography.X509Certificates.X509CertificateCollection,System.Security.Authentication.SslProtocols,bool)
                            at System.Net.Security.SslStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at WebSocketSharp.WebSocket.setClientStream () [0x000e1] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                             --- End of inner exception stack trace ---
                            at WebSocketSharp.WebSocket.setClientStream () [0x000fb] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.doHandshake () [0x00000] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.connect () [0x00073] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
21/08/2024 12:40:35|Debug|WebSocket.closeHandshake|Was clean?: False
                            sent: False
                            received: False
21/08/2024 12:40:35|Debug|WebSocket.sendHttpRequest|A request to the server:
                          GET /?userId=66aa283a7093a0984b7188e5 HTTP/1.1
                          User-Agent: websocket-sharp/1.0
                          Host: retstudio.io.vn
                          Upgrade: websocket
                          Connection: Upgrade
                          Sec-WebSocket-Key: HMK0p46/7IBtH8c50sIkiQ==
                          Sec-WebSocket-Version: 13
21/08/2024 12:40:35|Debug|WebSocket.sendHttpRequest|A response to this request:
                          HTTP/1.1 101 Switching Protocols
                          Date: Wed, 21 Aug 2024 05:40:35 GMT
                          Connection: upgrade
                          Upgrade: websocket
                          Sec-WebSocket-Accept: wk7ibcvudtD+UD5lt1MEFlWpYO4=
                          CF-Ray: 8b684a0b1d178502-HKG
                          CF-Cache-Status: DYNAMIC
                          Server: cloudflare
                          alt-svc: h3=":443"; ma=86400
21/08/2024 12:48:40|Info |WebSocket.close|The closing is already in progress.
21/08/2024 12:48:40|Debug|WebSocket.closeHandshake|Was clean?: True
                            sent: True
                            received: True
21/08/2024 12:48:40|Info |WebSocket.close|The connection has already been closed.
21/08/2024 12:48:40|Info |WebSocket.close|The connection has already been closed.
21/08/2024 12:52:53|Warn |Logger.set_Level|The current logging level has been changed to Debug.
21/08/2024 12:52:53|Fatal|WebSocket.connect|WebSocketSharp.WebSocketException: An error has occurred during a TLS handshake. ---> System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Mono.Security.Interface.TlsException: Handshake failed - error code: UNITYTLS_INTERNAL_ERROR, verify result: UNITYTLS_X509VERIFY_NOT_DONE
                            at Mono.Unity.Debug.CheckAndThrow (Mono.Unity.UnityTls+unitytls_errorstate errorState, Mono.Unity.UnityTls+unitytls_x509verify_result verifyResult, System.String context, Mono.Security.Interface.AlertDescription defaultAlert) [0x00036] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Unity.UnityTlsContext.ProcessHandshake () [0x00082] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake (Mono.Net.Security.AsyncOperationStatus status, System.Boolean renegotiate) [0x000da] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake(Mono.Net.Security.AsyncOperationStatus,bool)
                            at Mono.Net.Security.AsyncHandshakeRequest.Run (Mono.Net.Security.AsyncOperationStatus status) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.AsyncProtocolRequest.ProcessOperation (System.Threading.CancellationToken cancellationToken) [0x000fc] in <e2f37d58276e431f93af9ff200e09912>:0 
                             --- End of inner exception stack trace ---
                            at Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x0004b] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient(string,System.Security.Cryptography.X509Certificates.X509CertificateCollection,System.Security.Authentication.SslProtocols,bool)
                            at System.Net.Security.SslStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at WebSocketSharp.WebSocket.setClientStream () [0x000e1] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                             --- End of inner exception stack trace ---
                            at WebSocketSharp.WebSocket.setClientStream () [0x000fb] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.doHandshake () [0x00000] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.connect () [0x00073] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
21/08/2024 12:52:53|Debug|WebSocket.closeHandshake|Was clean?: False
                            sent: False
                            received: False
21/08/2024 12:52:54|Debug|WebSocket.sendHttpRequest|A request to the server:
                          GET /?userId=66aa283a7093a0984b7188e5 HTTP/1.1
                          User-Agent: websocket-sharp/1.0
                          Host: retstudio.io.vn
                          Upgrade: websocket
                          Connection: Upgrade
                          Sec-WebSocket-Key: QV/zZETAxr9G2z1bw3r43Q==
                          Sec-WebSocket-Version: 13
21/08/2024 12:52:54|Debug|WebSocket.sendHttpRequest|A response to this request:
                          HTTP/1.1 101 Switching Protocols
                          Date: Wed, 21 Aug 2024 05:52:54 GMT
                          Connection: upgrade
                          Upgrade: websocket
                          Sec-WebSocket-Accept: JIXu/liOtBE+KpwiCXNMWLhgsN8=
                          CF-Ray: 8b685c12786e8502-HKG
                          CF-Cache-Status: DYNAMIC
                          Server: cloudflare
                          alt-svc: h3=":443"; ma=86400
21/08/2024 12:53:55|Info |WebSocket.close|The closing is already in progress.
21/08/2024 12:53:55|Debug|WebSocket.closeHandshake|Was clean?: True
                            sent: True
                            received: True
21/08/2024 12:53:55|Info |WebSocket.close|The connection has already been closed.
21/08/2024 12:53:55|Info |WebSocket.close|The connection has already been closed.
21/08/2024 12:57:02|Warn |Logger.set_Level|The current logging level has been changed to Debug.
21/08/2024 12:57:02|Fatal|WebSocket.connect|WebSocketSharp.WebSocketException: An error has occurred during a TLS handshake. ---> System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Mono.Security.Interface.TlsException: Handshake failed - error code: UNITYTLS_INTERNAL_ERROR, verify result: UNITYTLS_X509VERIFY_NOT_DONE
                            at Mono.Unity.Debug.CheckAndThrow (Mono.Unity.UnityTls+unitytls_errorstate errorState, Mono.Unity.UnityTls+unitytls_x509verify_result verifyResult, System.String context, Mono.Security.Interface.AlertDescription defaultAlert) [0x00036] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Unity.UnityTlsContext.ProcessHandshake () [0x00082] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake (Mono.Net.Security.AsyncOperationStatus status, System.Boolean renegotiate) [0x000da] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake(Mono.Net.Security.AsyncOperationStatus,bool)
                            at Mono.Net.Security.AsyncHandshakeRequest.Run (Mono.Net.Security.AsyncOperationStatus status) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.AsyncProtocolRequest.ProcessOperation (System.Threading.CancellationToken cancellationToken) [0x000fc] in <e2f37d58276e431f93af9ff200e09912>:0 
                             --- End of inner exception stack trace ---
                            at Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x0004b] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient(string,System.Security.Cryptography.X509Certificates.X509CertificateCollection,System.Security.Authentication.SslProtocols,bool)
                            at System.Net.Security.SslStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at WebSocketSharp.WebSocket.setClientStream () [0x000e1] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                             --- End of inner exception stack trace ---
                            at WebSocketSharp.WebSocket.setClientStream () [0x000fb] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.doHandshake () [0x00000] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.connect () [0x00073] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
21/08/2024 12:57:02|Debug|WebSocket.closeHandshake|Was clean?: False
                            sent: False
                            received: False
21/08/2024 12:57:02|Debug|WebSocket.sendHttpRequest|A request to the server:
                          GET /?userId=66aa283a7093a0984b7188e5 HTTP/1.1
                          User-Agent: websocket-sharp/1.0
                          Host: retstudio.io.vn
                          Upgrade: websocket
                          Connection: Upgrade
                          Sec-WebSocket-Key: PwlCoG94JhHE+Sid57yerw==
                          Sec-WebSocket-Version: 13
21/08/2024 12:57:02|Debug|WebSocket.sendHttpRequest|A response to this request:
                          HTTP/1.1 101 Switching Protocols
                          Date: Wed, 21 Aug 2024 05:57:02 GMT
                          Connection: upgrade
                          Upgrade: websocket
                          Sec-WebSocket-Accept: uSE5hfWCNXX/ePFLz9LxZ2DDY7I=
                          CF-Ray: 8b6862225c3a8502-HKG
                          CF-Cache-Status: DYNAMIC
                          Server: cloudflare
                          alt-svc: h3=":443"; ma=86400
21/08/2024 12:58:04|Info |WebSocket.close|The closing is already in progress.
21/08/2024 12:58:04|Debug|WebSocket.closeHandshake|Was clean?: True
                            sent: True
                            received: True
21/08/2024 12:58:04|Info |WebSocket.close|The connection has already been closed.
21/08/2024 12:58:04|Info |WebSocket.close|The connection has already been closed.
21/08/2024 12:58:14|Warn |Logger.set_Level|The current logging level has been changed to Debug.
21/08/2024 12:58:14|Fatal|WebSocket.connect|WebSocketSharp.WebSocketException: An error has occurred during a TLS handshake. ---> System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Mono.Security.Interface.TlsException: Handshake failed - error code: UNITYTLS_INTERNAL_ERROR, verify result: UNITYTLS_X509VERIFY_NOT_DONE
                            at Mono.Unity.Debug.CheckAndThrow (Mono.Unity.UnityTls+unitytls_errorstate errorState, Mono.Unity.UnityTls+unitytls_x509verify_result verifyResult, System.String context, Mono.Security.Interface.AlertDescription defaultAlert) [0x00036] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Unity.UnityTlsContext.ProcessHandshake () [0x00082] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake (Mono.Net.Security.AsyncOperationStatus status, System.Boolean renegotiate) [0x000da] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake(Mono.Net.Security.AsyncOperationStatus,bool)
                            at Mono.Net.Security.AsyncHandshakeRequest.Run (Mono.Net.Security.AsyncOperationStatus status) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.AsyncProtocolRequest.ProcessOperation (System.Threading.CancellationToken cancellationToken) [0x000fc] in <e2f37d58276e431f93af9ff200e09912>:0 
                             --- End of inner exception stack trace ---
                            at Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x0004b] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient(string,System.Security.Cryptography.X509Certificates.X509CertificateCollection,System.Security.Authentication.SslProtocols,bool)
                            at System.Net.Security.SslStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at WebSocketSharp.WebSocket.setClientStream () [0x000e1] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                             --- End of inner exception stack trace ---
                            at WebSocketSharp.WebSocket.setClientStream () [0x000fb] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.doHandshake () [0x00000] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.connect () [0x00073] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
21/08/2024 12:58:14|Debug|WebSocket.closeHandshake|Was clean?: False
                            sent: False
                            received: False
21/08/2024 12:58:15|Debug|WebSocket.sendHttpRequest|A request to the server:
                          GET /?userId=66aa283a7093a0984b7188e5 HTTP/1.1
                          User-Agent: websocket-sharp/1.0
                          Host: retstudio.io.vn
                          Upgrade: websocket
                          Connection: Upgrade
                          Sec-WebSocket-Key: 6YjRsZK7I01cHQ33cC45hQ==
                          Sec-WebSocket-Version: 13
21/08/2024 12:58:15|Debug|WebSocket.sendHttpRequest|A response to this request:
                          HTTP/1.1 101 Switching Protocols
                          Date: Wed, 21 Aug 2024 05:58:15 GMT
                          Connection: upgrade
                          Upgrade: websocket
                          Sec-WebSocket-Accept: ynq2AyQ3fkMAGbh1M2zSwPAp+C0=
                          CF-Ray: 8b6863e8d8008502-HKG
                          CF-Cache-Status: DYNAMIC
                          Server: cloudflare
                          alt-svc: h3=":443"; ma=86400
21/08/2024 13:07:54|Info |WebSocket.close|The closing is already in progress.
21/08/2024 13:07:54|Debug|WebSocket.closeHandshake|Was clean?: True
                            sent: True
                            received: True
21/08/2024 13:07:54|Info |WebSocket.close|The connection has already been closed.
21/08/2024 13:07:54|Info |WebSocket.close|The connection has already been closed.
21/08/2024 13:08:32|Warn |Logger.set_Level|The current logging level has been changed to Debug.
21/08/2024 13:08:32|Fatal|WebSocket.connect|WebSocketSharp.WebSocketException: An error has occurred during a TLS handshake. ---> System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Mono.Security.Interface.TlsException: Handshake failed - error code: UNITYTLS_INTERNAL_ERROR, verify result: UNITYTLS_X509VERIFY_NOT_DONE
                            at Mono.Unity.Debug.CheckAndThrow (Mono.Unity.UnityTls+unitytls_errorstate errorState, Mono.Unity.UnityTls+unitytls_x509verify_result verifyResult, System.String context, Mono.Security.Interface.AlertDescription defaultAlert) [0x00036] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Unity.UnityTlsContext.ProcessHandshake () [0x00082] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake (Mono.Net.Security.AsyncOperationStatus status, System.Boolean renegotiate) [0x000da] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake(Mono.Net.Security.AsyncOperationStatus,bool)
                            at Mono.Net.Security.AsyncHandshakeRequest.Run (Mono.Net.Security.AsyncOperationStatus status) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.AsyncProtocolRequest.ProcessOperation (System.Threading.CancellationToken cancellationToken) [0x000fc] in <e2f37d58276e431f93af9ff200e09912>:0 
                             --- End of inner exception stack trace ---
                            at Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x0004b] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient(string,System.Security.Cryptography.X509Certificates.X509CertificateCollection,System.Security.Authentication.SslProtocols,bool)
                            at System.Net.Security.SslStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at WebSocketSharp.WebSocket.setClientStream () [0x000e1] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                             --- End of inner exception stack trace ---
                            at WebSocketSharp.WebSocket.setClientStream () [0x000fb] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.doHandshake () [0x00000] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.connect () [0x00073] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
21/08/2024 13:08:32|Debug|WebSocket.closeHandshake|Was clean?: False
                            sent: False
                            received: False
21/08/2024 13:08:32|Debug|WebSocket.sendHttpRequest|A request to the server:
                          GET /?userId=66aa283a7093a0984b7188e5 HTTP/1.1
                          User-Agent: websocket-sharp/1.0
                          Host: retstudio.io.vn
                          Upgrade: websocket
                          Connection: Upgrade
                          Sec-WebSocket-Key: +KWDEvpeONNrFIWFT2rSMA==
                          Sec-WebSocket-Version: 13
21/08/2024 13:08:34|Debug|WebSocket.sendHttpRequest|A response to this request:
                          HTTP/1.1 101 Switching Protocols
                          Date: Wed, 21 Aug 2024 06:08:34 GMT
                          Connection: upgrade
                          Upgrade: websocket
                          Sec-WebSocket-Accept: L+7t6YgWFCZ9bEklqBR6iiWjno0=
                          CF-Ray: 8b6873041eab8502-HKG
                          CF-Cache-Status: DYNAMIC
                          Server: cloudflare
                          alt-svc: h3=":443"; ma=86400
21/08/2024 13:15:21|Info |WebSocket.close|The closing is already in progress.
21/08/2024 13:15:21|Debug|WebSocket.closeHandshake|Was clean?: True
                            sent: True
                            received: True
21/08/2024 13:15:21|Info |WebSocket.close|The connection has already been closed.
21/08/2024 13:15:21|Info |WebSocket.close|The connection has already been closed.
21/08/2024 13:15:49|Warn |Logger.set_Level|The current logging level has been changed to Debug.
21/08/2024 13:15:49|Fatal|WebSocket.connect|WebSocketSharp.WebSocketException: An error has occurred during a TLS handshake. ---> System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Mono.Security.Interface.TlsException: Handshake failed - error code: UNITYTLS_INTERNAL_ERROR, verify result: UNITYTLS_X509VERIFY_NOT_DONE
                            at Mono.Unity.Debug.CheckAndThrow (Mono.Unity.UnityTls+unitytls_errorstate errorState, Mono.Unity.UnityTls+unitytls_x509verify_result verifyResult, System.String context, Mono.Security.Interface.AlertDescription defaultAlert) [0x00036] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Unity.UnityTlsContext.ProcessHandshake () [0x00082] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake (Mono.Net.Security.AsyncOperationStatus status, System.Boolean renegotiate) [0x000da] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake(Mono.Net.Security.AsyncOperationStatus,bool)
                            at Mono.Net.Security.AsyncHandshakeRequest.Run (Mono.Net.Security.AsyncOperationStatus status) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.AsyncProtocolRequest.ProcessOperation (System.Threading.CancellationToken cancellationToken) [0x000fc] in <e2f37d58276e431f93af9ff200e09912>:0 
                             --- End of inner exception stack trace ---
                            at Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x0004b] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient(string,System.Security.Cryptography.X509Certificates.X509CertificateCollection,System.Security.Authentication.SslProtocols,bool)
                            at System.Net.Security.SslStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at WebSocketSharp.WebSocket.setClientStream () [0x000e1] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                             --- End of inner exception stack trace ---
                            at WebSocketSharp.WebSocket.setClientStream () [0x000fb] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.doHandshake () [0x00000] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.connect () [0x00073] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
21/08/2024 13:15:49|Debug|WebSocket.closeHandshake|Was clean?: False
                            sent: False
                            received: False
21/08/2024 13:15:49|Debug|WebSocket.sendHttpRequest|A request to the server:
                          GET /?userId=66aa283a7093a0984b7188e5 HTTP/1.1
                          User-Agent: websocket-sharp/1.0
                          Host: retstudio.io.vn
                          Upgrade: websocket
                          Connection: Upgrade
                          Sec-WebSocket-Key: e4j9Y6nBevSo/Gm9L0G0pQ==
                          Sec-WebSocket-Version: 13
21/08/2024 13:15:49|Debug|WebSocket.sendHttpRequest|A response to this request:
                          HTTP/1.1 101 Switching Protocols
                          Date: Wed, 21 Aug 2024 06:15:50 GMT
                          Connection: upgrade
                          Upgrade: websocket
                          Sec-WebSocket-Accept: WEuwWjTGgFT8pWcor8cwSvJnVsY=
                          CF-Ray: 8b687da8da728502-HKG
                          CF-Cache-Status: DYNAMIC
                          Server: cloudflare
                          alt-svc: h3=":443"; ma=86400
21/08/2024 13:16:18|Info |WebSocket.close|The closing is already in progress.
21/08/2024 13:16:18|Debug|WebSocket.closeHandshake|Was clean?: True
                            sent: True
                            received: True
21/08/2024 13:16:18|Info |WebSocket.close|The connection has already been closed.
21/08/2024 13:16:18|Info |WebSocket.close|The connection has already been closed.
21/08/2024 13:16:37|Warn |Logger.set_Level|The current logging level has been changed to Debug.
21/08/2024 13:16:37|Fatal|WebSocket.connect|WebSocketSharp.WebSocketException: An error has occurred during a TLS handshake. ---> System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Mono.Security.Interface.TlsException: Handshake failed - error code: UNITYTLS_INTERNAL_ERROR, verify result: UNITYTLS_X509VERIFY_NOT_DONE
                            at Mono.Unity.Debug.CheckAndThrow (Mono.Unity.UnityTls+unitytls_errorstate errorState, Mono.Unity.UnityTls+unitytls_x509verify_result verifyResult, System.String context, Mono.Security.Interface.AlertDescription defaultAlert) [0x00036] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Unity.UnityTlsContext.ProcessHandshake () [0x00082] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake (Mono.Net.Security.AsyncOperationStatus status, System.Boolean renegotiate) [0x000da] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake(Mono.Net.Security.AsyncOperationStatus,bool)
                            at Mono.Net.Security.AsyncHandshakeRequest.Run (Mono.Net.Security.AsyncOperationStatus status) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.AsyncProtocolRequest.ProcessOperation (System.Threading.CancellationToken cancellationToken) [0x000fc] in <e2f37d58276e431f93af9ff200e09912>:0 
                             --- End of inner exception stack trace ---
                            at Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x0004b] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient(string,System.Security.Cryptography.X509Certificates.X509CertificateCollection,System.Security.Authentication.SslProtocols,bool)
                            at System.Net.Security.SslStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at WebSocketSharp.WebSocket.setClientStream () [0x000e1] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                             --- End of inner exception stack trace ---
                            at WebSocketSharp.WebSocket.setClientStream () [0x000fb] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.doHandshake () [0x00000] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.connect () [0x00073] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
21/08/2024 13:16:37|Debug|WebSocket.closeHandshake|Was clean?: False
                            sent: False
                            received: False
21/08/2024 13:16:37|Debug|WebSocket.sendHttpRequest|A request to the server:
                          GET /?userId=66aa283a7093a0984b7188e5 HTTP/1.1
                          User-Agent: websocket-sharp/1.0
                          Host: retstudio.io.vn
                          Upgrade: websocket
                          Connection: Upgrade
                          Sec-WebSocket-Key: pG9utKdB8i/TtCcwYxWKGg==
                          Sec-WebSocket-Version: 13
21/08/2024 13:16:37|Debug|WebSocket.sendHttpRequest|A response to this request:
                          HTTP/1.1 101 Switching Protocols
                          Date: Wed, 21 Aug 2024 06:16:37 GMT
                          Connection: upgrade
                          Upgrade: websocket
                          Sec-WebSocket-Accept: b9ylOrAS/pffcu6uwr/5MP2MwfA=
                          CF-Ray: 8b687ed3987f8502-HKG
                          CF-Cache-Status: DYNAMIC
                          Server: cloudflare
                          alt-svc: h3=":443"; ma=86400
21/08/2024 13:36:03|Info |WebSocket.close|The closing is already in progress.
21/08/2024 13:36:03|Debug|WebSocket.closeHandshake|Was clean?: True
                            sent: True
                            received: True
21/08/2024 13:36:03|Info |WebSocket.close|The connection has already been closed.
21/08/2024 13:36:03|Info |WebSocket.close|The connection has already been closed.
21/08/2024 13:46:42|Warn |Logger.set_Level|The current logging level has been changed to Debug.
21/08/2024 13:46:42|Fatal|WebSocket.connect|WebSocketSharp.WebSocketException: An error has occurred during a TLS handshake. ---> System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Mono.Security.Interface.TlsException: Handshake failed - error code: UNITYTLS_INTERNAL_ERROR, verify result: UNITYTLS_X509VERIFY_NOT_DONE
                            at Mono.Unity.Debug.CheckAndThrow (Mono.Unity.UnityTls+unitytls_errorstate errorState, Mono.Unity.UnityTls+unitytls_x509verify_result verifyResult, System.String context, Mono.Security.Interface.AlertDescription defaultAlert) [0x00036] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Unity.UnityTlsContext.ProcessHandshake () [0x00082] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake (Mono.Net.Security.AsyncOperationStatus status, System.Boolean renegotiate) [0x000da] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake(Mono.Net.Security.AsyncOperationStatus,bool)
                            at Mono.Net.Security.AsyncHandshakeRequest.Run (Mono.Net.Security.AsyncOperationStatus status) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.AsyncProtocolRequest.ProcessOperation (System.Threading.CancellationToken cancellationToken) [0x000fc] in <e2f37d58276e431f93af9ff200e09912>:0 
                             --- End of inner exception stack trace ---
                            at Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x0004b] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient(string,System.Security.Cryptography.X509Certificates.X509CertificateCollection,System.Security.Authentication.SslProtocols,bool)
                            at System.Net.Security.SslStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at WebSocketSharp.WebSocket.setClientStream () [0x000e1] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                             --- End of inner exception stack trace ---
                            at WebSocketSharp.WebSocket.setClientStream () [0x000fb] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.doHandshake () [0x00000] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.connect () [0x00073] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
21/08/2024 13:46:42|Debug|WebSocket.closeHandshake|Was clean?: False
                            sent: False
                            received: False
21/08/2024 13:46:42|Debug|WebSocket.sendHttpRequest|A request to the server:
                          GET /?userId=66aa283a7093a0984b7188e5 HTTP/1.1
                          User-Agent: websocket-sharp/1.0
                          Host: retstudio.io.vn
                          Upgrade: websocket
                          Connection: Upgrade
                          Sec-WebSocket-Key: Tbdoh7lTW+pmKKCAErPnow==
                          Sec-WebSocket-Version: 13
21/08/2024 13:46:43|Debug|WebSocket.sendHttpRequest|A response to this request:
                          HTTP/1.1 101 Switching Protocols
                          Date: Wed, 21 Aug 2024 06:46:43 GMT
                          Connection: upgrade
                          Upgrade: websocket
                          Sec-WebSocket-Accept: IXJwdFUsVnYluQ8eWciXeB7snFE=
                          CF-Ray: 8b68aae728ef8502-HKG
                          CF-Cache-Status: DYNAMIC
                          Server: cloudflare
                          alt-svc: h3=":443"; ma=86400
21/08/2024 14:05:26|Info |WebSocket.close|The closing is already in progress.
21/08/2024 14:05:26|Debug|WebSocket.closeHandshake|Was clean?: True
                            sent: True
                            received: True
21/08/2024 14:05:26|Info |WebSocket.close|The connection has already been closed.
21/08/2024 14:05:26|Info |WebSocket.close|The connection has already been closed.
21/08/2024 14:31:32|Warn |Logger.set_Level|The current logging level has been changed to Debug.
21/08/2024 14:31:32|Fatal|WebSocket.connect|WebSocketSharp.WebSocketException: An error has occurred during a TLS handshake. ---> System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Mono.Security.Interface.TlsException: Handshake failed - error code: UNITYTLS_INTERNAL_ERROR, verify result: UNITYTLS_X509VERIFY_NOT_DONE
                            at Mono.Unity.Debug.CheckAndThrow (Mono.Unity.UnityTls+unitytls_errorstate errorState, Mono.Unity.UnityTls+unitytls_x509verify_result verifyResult, System.String context, Mono.Security.Interface.AlertDescription defaultAlert) [0x00036] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Unity.UnityTlsContext.ProcessHandshake () [0x00082] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake (Mono.Net.Security.AsyncOperationStatus status, System.Boolean renegotiate) [0x000da] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake(Mono.Net.Security.AsyncOperationStatus,bool)
                            at Mono.Net.Security.AsyncHandshakeRequest.Run (Mono.Net.Security.AsyncOperationStatus status) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at Mono.Net.Security.AsyncProtocolRequest.ProcessOperation (System.Threading.CancellationToken cancellationToken) [0x000fc] in <e2f37d58276e431f93af9ff200e09912>:0 
                             --- End of inner exception stack trace ---
                            at Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x0004b] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient(string,System.Security.Cryptography.X509Certificates.X509CertificateCollection,System.Security.Authentication.SslProtocols,bool)
                            at System.Net.Security.SslStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                            at WebSocketSharp.WebSocket.setClientStream () [0x000e1] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                             --- End of inner exception stack trace ---
                            at WebSocketSharp.WebSocket.setClientStream () [0x000fb] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.doHandshake () [0x00000] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                            at WebSocketSharp.WebSocket.connect () [0x00073] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
21/08/2024 14:31:32|Debug|WebSocket.closeHandshake|Was clean?: False
                            sent: False
                            received: False
21/08/2024 14:31:33|Debug|WebSocket.sendHttpRequest|A request to the server:
                          GET /?userId=66aa283a7093a0984b7188e5 HTTP/1.1
                          User-Agent: websocket-sharp/1.0
                          Host: retstudio.io.vn
                          Upgrade: websocket
                          Connection: Upgrade
                          Sec-WebSocket-Key: 5czUJDuTBB4/FIUQtD6FCg==
                          Sec-WebSocket-Version: 13
21/08/2024 14:31:33|Debug|WebSocket.sendHttpRequest|A response to this request:
                          HTTP/1.1 101 Switching Protocols
                          Date: Wed, 21 Aug 2024 07:31:33 GMT
                          Connection: upgrade
                          Upgrade: websocket
                          Sec-WebSocket-Accept: hZ5YSaMDVFGA64tcFF75Q+kpT+Y=
                          CF-Ray: 8b68ec961d308502-HKG
                          CF-Cache-Status: DYNAMIC
                          Server: cloudflare
                          alt-svc: h3=":443"; ma=86400
21/08/2024 14:37:18|Info |WebSocket.close|The closing is already in progress.
21/08/2024 14:37:18|Debug|WebSocket.closeHandshake|Was clean?: True
                            sent: True
                            received: True
21/08/2024 14:37:18|Info |WebSocket.close|The connection has already been closed.
21/08/2024 14:37:18|Info |WebSocket.close|The connection has already been closed.
8/21/2024 7:40:32 PM|Warn |Logger.set_Level|The current logging level has been changed to Debug.
8/21/2024 7:40:32 PM|Fatal|WebSocket.connect|WebSocketSharp.WebSocketException: An error has occurred during a TLS handshake. ---> System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Mono.Security.Interface.TlsException: Handshake failed - error code: UNITYTLS_INTERNAL_ERROR, verify result: UNITYTLS_X509VERIFY_NOT_DONE
                             at Mono.Unity.Debug.CheckAndThrow (Mono.Unity.UnityTls+unitytls_errorstate errorState, Mono.Unity.UnityTls+unitytls_x509verify_result verifyResult, System.String context, Mono.Security.Interface.AlertDescription defaultAlert) [0x00036] in <e2f37d58276e431f93af9ff200e09912>:0 
                             at Mono.Unity.UnityTlsContext.ProcessHandshake () [0x00082] in <e2f37d58276e431f93af9ff200e09912>:0 
                             at Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake (Mono.Net.Security.AsyncOperationStatus status, System.Boolean renegotiate) [0x000da] in <e2f37d58276e431f93af9ff200e09912>:0 
                             at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake(Mono.Net.Security.AsyncOperationStatus,bool)
                             at Mono.Net.Security.AsyncHandshakeRequest.Run (Mono.Net.Security.AsyncOperationStatus status) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                             at Mono.Net.Security.AsyncProtocolRequest.ProcessOperation (System.Threading.CancellationToken cancellationToken) [0x000fc] in <e2f37d58276e431f93af9ff200e09912>:0 
                              --- End of inner exception stack trace ---
                             at Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x0004b] in <e2f37d58276e431f93af9ff200e09912>:0 
                             at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient(string,System.Security.Cryptography.X509Certificates.X509CertificateCollection,System.Security.Authentication.SslProtocols,bool)
                             at System.Net.Security.SslStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                             at WebSocketSharp.WebSocket.setClientStream () [0x000e1] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                              --- End of inner exception stack trace ---
                             at WebSocketSharp.WebSocket.setClientStream () [0x000fb] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                             at WebSocketSharp.WebSocket.doHandshake () [0x00000] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                             at WebSocketSharp.WebSocket.connect () [0x00073] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
8/21/2024 7:40:32 PM|Debug|WebSocket.closeHandshake|Was clean?: False
                             sent: False
                             received: False
8/21/2024 7:40:32 PM|Debug|WebSocket.sendHttpRequest|A request to the server:
                           GET /?userId=66adc30c443e38de97ecdf0b HTTP/1.1
                           User-Agent: websocket-sharp/1.0
                           Host: retstudio.io.vn
                           Upgrade: websocket
                           Connection: Upgrade
                           Sec-WebSocket-Key: JA5bMgT/wUYfhDI4W3UiSw==
                           Sec-WebSocket-Version: 13
8/21/2024 7:40:32 PM|Debug|WebSocket.sendHttpRequest|A response to this request:
                           HTTP/1.1 101 Switching Protocols
                           Date: Wed, 21 Aug 2024 12:40:35 GMT
                           Connection: upgrade
                           Upgrade: websocket
                           Sec-WebSocket-Accept: 0x1Rl9S98qot9iHqNYKa6GEhkDA=
                           CF-Ray: 8b6ab141ca9a5dde-HKG
                           CF-Cache-Status: DYNAMIC
                           Server: cloudflare
                           alt-svc: h3=":443"; ma=86400
8/21/2024 7:41:31 PM|Info |WebSocket.close|The closing is already in progress.
8/21/2024 7:41:31 PM|Debug|WebSocket.closeHandshake|Was clean?: True
                             sent: True
                             received: True
8/21/2024 7:41:31 PM|Info |WebSocket.close|The connection has already been closed.
8/21/2024 7:41:31 PM|Info |WebSocket.close|The connection has already been closed.
8/21/2024 10:21:05 PM|Warn |Logger.set_Level|The current logging level has been changed to Debug.
8/21/2024 10:21:05 PM|Fatal|WebSocket.connect|WebSocketSharp.WebSocketException: An error has occurred during a TLS handshake. ---> System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Mono.Security.Interface.TlsException: Handshake failed - error code: UNITYTLS_INTERNAL_ERROR, verify result: UNITYTLS_X509VERIFY_NOT_DONE
                              at Mono.Unity.Debug.CheckAndThrow (Mono.Unity.UnityTls+unitytls_errorstate errorState, Mono.Unity.UnityTls+unitytls_x509verify_result verifyResult, System.String context, Mono.Security.Interface.AlertDescription defaultAlert) [0x00036] in <e2f37d58276e431f93af9ff200e09912>:0 
                              at Mono.Unity.UnityTlsContext.ProcessHandshake () [0x00082] in <e2f37d58276e431f93af9ff200e09912>:0 
                              at Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake (Mono.Net.Security.AsyncOperationStatus status, System.Boolean renegotiate) [0x000da] in <e2f37d58276e431f93af9ff200e09912>:0 
                              at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.ProcessHandshake(Mono.Net.Security.AsyncOperationStatus,bool)
                              at Mono.Net.Security.AsyncHandshakeRequest.Run (Mono.Net.Security.AsyncOperationStatus status) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                              at Mono.Net.Security.AsyncProtocolRequest.ProcessOperation (System.Threading.CancellationToken cancellationToken) [0x000fc] in <e2f37d58276e431f93af9ff200e09912>:0 
                               --- End of inner exception stack trace ---
                              at Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x0004b] in <e2f37d58276e431f93af9ff200e09912>:0 
                              at (wrapper remoting-invoke-with-check) Mono.Net.Security.MobileAuthenticatedStream.AuthenticateAsClient(string,System.Security.Cryptography.X509Certificates.X509CertificateCollection,System.Security.Authentication.SslProtocols,bool)
                              at System.Net.Security.SslStream.AuthenticateAsClient (System.String targetHost, System.Security.Cryptography.X509Certificates.X509CertificateCollection clientCertificates, System.Security.Authentication.SslProtocols enabledSslProtocols, System.Boolean checkCertificateRevocation) [0x00006] in <e2f37d58276e431f93af9ff200e09912>:0 
                              at WebSocketSharp.WebSocket.setClientStream () [0x000e1] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                               --- End of inner exception stack trace ---
                              at WebSocketSharp.WebSocket.setClientStream () [0x000fb] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                              at WebSocketSharp.WebSocket.doHandshake () [0x00000] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
                              at WebSocketSharp.WebSocket.connect () [0x00073] in <3dbc6ef0559a4023b6744a59e39dceea>:0 
8/21/2024 10:21:05 PM|Debug|WebSocket.closeHandshake|Was clean?: False
                              sent: False
                              received: False
8/21/2024 10:21:05 PM|Debug|WebSocket.sendHttpRequest|A request to the server:
                            GET /?userId=66adc30c443e38de97ecdf0b HTTP/1.1
                            User-Agent: websocket-sharp/1.0
                            Host: retstudio.io.vn
                            Upgrade: websocket
                            Connection: Upgrade
                            Sec-WebSocket-Key: m3boWVUaF3W9CCyNKjQDsw==
                            Sec-WebSocket-Version: 13
8/21/2024 10:21:06 PM|Debug|WebSocket.sendHttpRequest|A response to this request:
                            HTTP/1.1 101 Switching Protocols
                            Date: Wed, 21 Aug 2024 15:21:08 GMT
                            Connection: upgrade
                            Upgrade: websocket
                            Sec-WebSocket-Accept: eSMWEP54V79jrelpQ85jE8FM0oA=
                            CF-Ray: 8b6b9c728c355dea-HKG
                            CF-Cache-Status: DYNAMIC
                            Server: cloudflare
                            alt-svc: h3=":443"; ma=86400
8/21/2024 10:21:58 PM|Info |WebSocket.close|The closing is already in progress.
8/21/2024 10:21:58 PM|Debug|WebSocket.closeHandshake|Was clean?: True
                              sent: True
                              received: True
8/21/2024 10:21:59 PM|Debug|WebSocket.sendHttpRequest|A request to the server:
                            GET /?userId=66adc30c443e38de97ecdf0b HTTP/1.1
                            User-Agent: websocket-sharp/1.0
                            Host: retstudio.io.vn
                            Upgrade: websocket
                            Connection: Upgrade
                            Sec-WebSocket-Key: m3boWVUaF3W9CCyNKjQDsw==
                            Sec-WebSocket-Version: 13
8/21/2024 10:21:59 PM|Debug|WebSocket.sendHttpRequest|A response to this request:
                            HTTP/1.1 101 Switching Protocols
                            Date: Wed, 21 Aug 2024 15:22:01 GMT
                            Connection: upgrade
                            Upgrade: websocket
                            Sec-WebSocket-Accept: eSMWEP54V79jrelpQ85jE8FM0oA=
                            CF-Ray: 8b6b9dbfb848099d-HKG
                            CF-Cache-Status: DYNAMIC
                            Server: cloudflare
                            alt-svc: h3=":443"; ma=86400
8/21/2024 10:21:59 PM|Info |WebSocket.close|The closing is already in progress.
8/21/2024 10:21:59 PM|Debug|WebSocket.closeHandshake|Was clean?: True
                              sent: True
                              received: True
